packages/inter-protocol/test/swingsetTests/reserve/bootstrap-assetReserve-upgrade.js

```diff
@@ -271,6 +271,11 @@ export const buildRootObject = async () => {
 
       metricsRecord = await E(metrics).getUpdateSince();
 
+      // verify allocations
+      const allocations = await E(arLimitedFacet).getAllocations();
+      assert.equal(allocations.Moola.value, moola.make(100_000n).value);
+      assert.equal(allocations.Moola.brand, moola.make(100_000n).brand);
```

Comment:
It would be nice if we could use `assertAmountEquals` here, but we don't have **t**, instead:
```suggestion
      assert.equal(allocations.Moola.value, 100_000n);
      assert.equal(allocations.Moola.brand, moola.brand);
```